Dockerfile

```diff
@@ -34,3 +34,4 @@ ADD fetch_raw_data.py /app
 ADD fetch_recovered_data.py /app
 ADD fetch_flow_definitions.py /app
 ADD generate_outputs.py /app
+ADD generate_analysis_graphs.py /app
```

Comment:
I'm a little surprised this is the only change that was needed given our discussion offline? How does this deal with the Selenium dependency?

Reply:
Ah, I see this is in #92 - no trouble